cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/eventData/addedToCart.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+/* API Includes */
+var Logger = require('dw/system/Logger');
+var URLUtils = require('dw/web/URLUtils');
+var ProductMgr = require('dw/catalog/ProductMgr');
+
+/* Script Modules */
+var klaviyoUtils = require('*/cartridge/scripts/klaviyo/utils');
+var KLImageSize = klaviyoUtils.KLImageSize;
+
+
+// prepares data for "Added to Cart" event
+function getData(basket) {
+    var data;
+
+    try {
+        data = {};
+        var basketItems = basket.getProductLineItems().toArray();
+
+        data.event = klaviyoUtils.EVENT_NAMES.addedToCart;
```

Comment:
See comment above, but I propose we move the event name logic currently living in `trackEvent` here